client/netbird-entrypoint.sh

```
@@ -51,8 +51,8 @@ wait_for_daemon_startup() {
 }
 
 connect() {
-  info "running 'netbird up'..."
-  "${NETBIRD_BIN}" up
+  info "running: netbird up $@"
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)